app/models/admin_menu.rb

```diff
@@ -34,7 +37,7 @@ class AdminMenu
       item(name: "mods"),
       item(name: "moderator actions ads", controller: "moderator_actions"),
       item(name: "privileged reactions"),
-      # item(name: "interaction limits", controller: "" )
+      # item(name: "interaction limits")
```

Comment:
@vaidehijoshi this is just a placeholder for the route that I know is upcoming in the next few weeks (if someone else is working on it, it will be quick for the uncomment), I'll be [creating an issue as we discussed](https://github.com/forem/rfcs/pull/50#issuecomment-800113625) before the end of the week and so I think that should be sufficient.